docs/tutorial/t-create-client-credentials.md

```diff
@@ -0,0 +1,399 @@
+# Create Client Credentials
+
+After having succesfully deployed a Kafka cluster you most probably would like to have client applications to connect to this cluster.
+
+## Install required software
+
+### Keytool (Java)
+
+If your clients applications are based on Java, consider that it might need to use special Java file formats for the certificates called keystore and trustore.
+
+To generate these files we will need `keytool` binary. This commes preinstalled with any Java installation.
+
+Make sure you have Java installed and keytool was installed with it.
+
```

Comment:
**todo:** Give example of how to install Java on an ubuntu machine.

e.g `apt update && apt install openjdk-11-jre-headless`